packages/google_maps_flutter/google_maps_flutter_android/android/src/test/java/io/flutter/plugins/googlemaps/GoogleMapControllerTest.java

```diff
@@ -145,4 +148,22 @@ public void MethodCalledAfterControllerIsDestroyed() throws InterruptedException
     argument.getValue().onMapLoaded();
     verify(mapView, never()).invalidate();
   }
+
+  @Test
+  public void OnMapReadySetsPaddingIfInitialPaddingIsThere() {
+    float padding = 10f;
+    int paddingWithDensity = (int) (10f * googleMapController.density);
```

Comment:
Is this 10f the same as the padding being set in the line above? Or is it a random value? If it is random can you change it to something other than the value being used for padding?

Reply:
this should have been padding * googleMapController.density